mslib/msui/mscolab.py

```diff
@@ -359,32 +353,30 @@ def login_handler(self):
             "email": self.loginEmailLe.text(),
             "password": self.loginPasswordLe.text()
         }
-        s = requests.Session()
-        s.auth = self.auth
-        s.headers.update({'x-test': 'true'})
+        session = requests.Session()
+        session.auth = self.auth
+        session.headers.update({'x-test': 'true'})
         url = urljoin(self.mscolab_server_url, "token")
         url_recover_password = urljoin(self.mscolab_server_url, "reset_request")
         try:
-            r = s.post(url, data=data, timeout=tuple(config_loader(dataset="MSCOLAB_timeout")))
-            if r.status_code == 401:
-                raise requests.exceptions.ConnectionError
+            response = session.post(url, data=data, timeout=tuple(config_loader(dataset="MSCOLAB_timeout")))
+            response.raise_for_status()
```

Comment:
I am not sure what happens on e.g. 403 now

Reply:
interesting read 
https://stackoverflow.com/questions/61463224/when-to-use-raise-for-status-vs-status-code-testing

Reply:
I mean we only raised explicity on 401 raise requests.exceptions.ConnectionError
I have not found where it was used and got changed there now too

Reply:
The next envisioned change goes deeper in this direction, but I wanted to keep it "small".